app/components/windows/ChildWindow.vue.ts

```diff
@@ -85,8 +94,33 @@ export default class ChildWindow extends Vue {
     // that will do a bunch of synchronous IO.
     clearTimeout(this.refreshingTimeout);
     this.refreshingTimeout = window.setTimeout(() => {
-      this.components.push({ name: options.componentName, isShown: true, title: options.title });
+      this.components.push({
+        name: options.componentName,
+        isShown: true,
+        title: options.title,
+        hideStyleBlockers: options.hideStyleBlockers,
+      });
       this.setWindowTitle();
+      window.addEventListener('resize', this.windowSizeHandler);
```

Comment:
It seems a little complex to be adding and removing the resize listener every time we mount and unmount a component to the window.  Is the a reason we can't just have the the resize handler mount when the `ChildWindow` component mounts and just keep it there?

Reply:
using the vue lifecycle methods was giving me some extremely weird bugs when i was testing that out, so this was the closest compromise i could find

Reply:
Hmm.  Could be related to the fact that we are hiding the window without destroying it.